README.md

```diff
@@ -219,6 +220,10 @@ replicas and the one `hello-osiris` pod should be terminated.
 Make a request again, and watch as Osiris scales the deployment back to one
 replica and your request is handled successfully.
 
+You can also do the same with the 
+[hello-osiris-statefulset.yaml](example/hello-osiris-statefulset.yaml) example, 
+which uses a Kubernetes StatefulSet instead of a Deployment.
+
```

Comment:
I'm a little iffy on introducing a second example involving a statefulset since the underlying application isn't stateful. It's certainly not that it won't work. I'm just weighing value against maintenance burden.

Reply:
ok, I just removed the statefulset example